packages/conventional-changelog-core/lib/merge-config.js

```diff
@@ -251,6 +251,16 @@ function mergeConfig (options, context, gitRawCommitsOpts, parserOpts, writerOpt
         gitRawCommitsOpts.reverse = gitRawCommitsOpts.reverse || true
       }
 
+      if (options.mergeCommitFilter) {
+        if (options.mergeCommitFilter === 'exclude') {
+          gitRawCommitsOpts.merges = false
```

Comment:
If one would set `gitRawCommitsOpts.merges` to `null` or `true`  it would get overwritten to `false`.

Reply:
I think it should be moved to `conventional-changelog-cli`, as people using `conventional-changelog-core` directly can set `gitRawCommitsOpts.merges` and this is only needed for command line flags.

Reply:
got it. I will remove it from the core then